src/components/tables/Table.tsx

```diff
@@ -55,7 +55,7 @@ export default function Table<T extends object = {}>(
 
   return (
     <>
-      <div className="relative overflow-hidden rounded-md">
```

Comment:
this avoided tooltips to render properly so removing (having seen any side effect)